### DISCUSSION TITLE
Can content.canvas_reading support URL patterns?

### DISCUSSION THREAD

**Discussion by crocket** [qutebrowser/qutebrowser#7170] [Q&A] [answered]
content.webgl supports URL patterns. Some websites need webgl and canvas. Most don't.

**Comment by The-Compiler** [comment, marked as answer]
This isn't possible at the moment, see [[QTBUG-84486] Add setting attribute to disable reading from canvas - Qt Bug Tracker](https://bugreports.qt.io/browse/QTBUG-84486) and #5439.
